src/main/java/com/revature/studyforce/stacktrace/controller/SolutionController.java

```diff
@@ -14,7 +15,7 @@
  * @author Joshua Swanson
```

Comment:
We are following @link documentation above the author, controller methods, and service methods
EX:
@link SolutionService#findAll()